Sources/HipMobileUI/Helpers/Settings.cs

```diff
@@ -0,0 +1,43 @@
+// Helpers/Settings.cs
+using Plugin.Settings;
+using Plugin.Settings.Abstractions;
+
+namespace PaderbornUniversity.SILab.Hip.Mobile.UI.Helpers
+{
+  /// <summary>
+  /// This is the Settings static class that can be used in your Core solution or in any
+  /// of your client applications. All settings are laid out the same exact way with getters
+  /// and setters. 
+  /// </summary>
+  public static class Settings
```

Comment:
We already have such a class somewhere in our codebase, why did you add this here?

Reply:
This was a mistake. I will remove this file in the next commit